install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -55,8 +63,12 @@ initContainers:
     runAsGroup: 0
     runAsNonRoot: false
     runAsUser: 0
+  {{- else }}
+    runAsGroup: 1337
```

Comment:
Not including all the fields in securityContext in this case is a regression. All securityContext fields must be set in every case. Cf. #17318.

Reply:
get it. Updating all the inject templates